package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@absmach/magistrala-sdk",
-  "version": "0.3.12",
+  "version": "0.3.14",
```

Comment:
@ianmuchyri Why did we skip `0.3.13`?

Reply:
I skipped it accidentally. But now since this `0.3.14` was already published, I don't know if we can be able go lower

Reply:
Let's unpublish 0.3.14.

Reply:
We may not be able to publish under `0.3.13` in a day or so because it's been unpublished recently.

Reply:
Okay

Reply:
I will revert to `0.3.12` then

Reply:
I have reverted